pkg/types/installconfig.go

```diff
@@ -90,6 +100,11 @@ type InstallConfig struct {
 	// ImageContentSources lists sources/repositories for the release-image content.
 	// +optional
 	ImageContentSources []ImageContentSource `json:"imageContentSources,omitempty"`
+
+	// Publish controls how the user facing endpoints of the cluster like the k8s API, OpenShift routes etc. are exposed.
```

Comment:
nit: can we spell out `k8s` -> `Kubernetes` in docs like this?

Reply:
fixed with https://github.com/openshift/installer/compare/586800a7998182fd87bc00700adf24a49df216be...83adc04d6cf70ffc2f45bbe6a5e8d1d06173832d